app/app.go

```diff
@@ -285,7 +285,7 @@ func NewBandApp(
 			IBCKeeper:       app.IBCKeeper,
 			StakingKeeper:   app.StakingKeeper,
 			GlobalfeeKeeper: &app.GlobalFeeKeeper,
-			Lanes:           []*mempool.Lane{feedsLane, tssLane, oracleLane}, // every lane except default lane
+			Lanes:           []*mempool.Lane{feedsLane, tssLane, oracleReportLane}, // only lanes that are gas free
```

Comment:
should we merge branch from PR 633 into this brach and fix it? (open PR to that brach instead)